pyNN/nest/simulator.py

```diff
@@ -131,6 +134,45 @@ def _set_verbosity(self, verbosity):
         nest.set_verbosity('M_{}'.format(verbosity.upper()))
     verbosity = property(fset=_set_verbosity)
 
+    def set_status(self, nodes, params, val=None):
+        """
+        Wrapper around nest.SetStatus() to handle time offset
+        """
+        if self._time_offset == 0.0:
+            nest.SetStatus(nodes, params, val=val)
+        elif val is None:
+            if isinstance(params, list):
+                params_copy = []
+                for item in params:
+                    item_copy = {}
+                    for name, value in item.items():
+                        if name in ("start", "stop"):
```

Comment:
I've refactored this to remove the repetition (will push soon). It would be good to add additional relevant parameter names from the NEST models directory, but I think that can go in a separate PR.